roles/shiftstack/tasks/pre_test_shiftstack.yml

```diff
@@ -13,6 +13,13 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations
 # under the License.
+
+- name: Fail if the testconfig is not passed
+  ansible.builtin.fail:
+    msg: |
+      "The variable 'cifmw_run_test_shiftstack_testconfig' is not passed and it's required."
+  when: cifmw_run_test_shiftstack_testconfig is not defined
+
```

Comment:
This kind of task is usually done with `ansible.builtin.assert`. No need to use fail + when condition.

Reply:
Thanks, I changed it to an assert, which makes more sense indeed :)